From f9fe1f25f64b85a52b88f1bdbcf9de1b5b1b5070 Mon Sep 17 00:00:00 2001
From: zhang sanshan <b51434@freescale.com>
Date: Thu, 23 Jul 2015 09:49:20 +0800
Subject: [PATCH 516/602] MA-6915 [Android_MX7D]OTA: It can't build ota
 successfully sometimes.

The daily build sometimes can't build ota image successfully.
That was ota script did not find uboot-7d-touch.imx\uboot-7d-epdc.imx.

Signed-off-by: zhang sanshan <b51434@freescale.com>
---
 sabresd_7d/BoardConfig.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sabresd_7d/BoardConfig.mk b/sabresd_7d/BoardConfig.mk
index 5843084..de92faa 100644
--- a/sabresd_7d/BoardConfig.mk
+++ b/sabresd_7d/BoardConfig.mk
@@ -88,7 +88,7 @@ IMX_CAMERA_HAL_V2 := false
 TARGET_VSYNC_DIRECT_REFRESH := true
 
 BOARD_KERNEL_CMDLINE := console=ttymxc0,115200 init=/init androidboot.console=ttymxc0 consoleblank=0 androidboot.hardware=freescale 
-TARGET_BOOTLOADER_CONFIG := imx7d:mx7dsabresdandroid_config
+TARGET_BOOTLOADER_CONFIG := imx7d:mx7dsabresdandroid_config imx7d-touch:mx7dsabresdandroid_config imx7d-epdc:mx7dsabresdandroid_config
 TARGET_BOARD_DTS_CONFIG := imx7d:imx7d-sdb.dtb imx7d-touch:imx7d-sdb-touch.dtb imx7d-epdc:imx7d-sdb-epdc.dtb
 
 BOARD_SEPOLICY_DIRS := \
-- 
1.8.0

